src/hooks/useFetchPriceHistories/useFetchPriceHistories.ts

```diff
@@ -13,13 +14,20 @@ type UseFetchPriceHistories = (args: UseFetchPriceHistoriesArgs) => void
 
 export const useFetchPriceHistories: UseFetchPriceHistories = ({ assetIds, timeframe }) => {
   const dispatch = useAppDispatch()
+  const symbol = useAppSelector(selectSelectedCurrency)
+
+  const { findPriceHistoryByAssetId, findPriceHistoryByFiatSymbol } = marketApi.endpoints
   useEffect(
     () =>
       assetIds.forEach(assetId =>
-        dispatch(marketApi.endpoints.findPriceHistoryByAssetId.initiate({ assetId, timeframe })),
+        dispatch(findPriceHistoryByAssetId.initiate({ assetId, timeframe })),
       ),
     // assetIds ref changes, prevent infinite render
```

Comment:
No longer applicable? removal of `JSON.stringify` doesn't cause infinite re-renders?

Reply:
assetIds are now stable coming from the selector so no, `JSON.stringify` would have been a workaround prior